test/requests/path_completion_test.rb

```diff
@@ -35,7 +35,7 @@ def test_completion_command
       @store.set(uri: @uri, source: document.source, version: 1)
       RubyLsp::Executor.new(@store, @message_queue).execute({
         method: "textDocument/completion",
-        params: { textDocument: { uri: @uri }, position: end_position },
+        params: { textDocument: { uri: @uri.to_s }, position: end_position },
```

Comment:
(nit) Could we do the `to_s` conversion in `Executor#run` instead?

Reply:
We always receive the URIs as strings from the server, so the `Executor` takes them and turns them into `URI` objects.

We end up needing both for tests, but outside of tests we would never receive a uri parameter that isn't a string, so. I don't think it's worth adding logic to handle both cases just.

Reply:
Ah, true, it better reflects the implementation if we stick with the strings.